HousingSearchApi.Tests/HousingSearchApi.Tests.csproj

```diff
@@ -5,7 +5,7 @@
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
     <CodeAnalysisTreatWarningsAsErrors>true</CodeAnalysisTreatWarningsAsErrors>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Should we be turning this off? (Or are you just annoyed with as much as I am?)

Reply:
Just as annoyed :D I'll turn it on come next PR.